src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -69,11 +69,11 @@ const useCoreFeed = () => {
       };
       return color;
     }
-    const randomIndex = getRandomInt(0, listFeedColor.length);
+    const randomIndex = getRandomInt(0, listFeedColor.length - 1);
     let newColor = listFeedColor[randomIndex];
     newColor = {
       ...newColor,
-      bg: hexToRgb(newColor.bg, 0.25),
+      bg: hexToRgb(newColor?.bg, 0.25),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining (`newColor?.bg`) is a good practice to handle situations where `newColor.bg` might be null or undefined. However, it would be better to handle this case explicitly and provide a default value when `newColor.bg` is null or undefined. This can help avoid unexpected behavior in the `hexToRgb` function.
```diff
-      bg: hexToRgb(newColor?.bg, 0.25),
+      bg: newColor?.bg ? hexToRgb(newColor.bg, 0.25) : DEFAULT_COLOR,
```
In the above code, `DEFAULT_COLOR` should be a predefined constant representing a default color value.


<!-- This is an auto-generated comment by OSS CodeRabbit -->